### DISCUSSION TITLE
How do i setup the characters offset correctly?

### DISCUSSION THREAD

**Discussion by ryi3r** [Leather128/LeathersFunkinEngine#33] [Q&A]
So i've been using animation debug then editing the offsets.txt file to set it and it doesn't work, can someone help me to know how to set the offsets correctly?

**Comment by Leather128** [comment]
Are you using a compiled version of the game? If so, make sure that you either, are changing the files in the bin folder, or are recompiling after changing the files.

**Comment by ryi3r** [comment]
I'm using the source code, compiling it myself, plus i've deleted the assets folder to check if it re-compiled that folder.

**Reply by Leather128** [reply]
So you changed the `offsets.txt` file in the `source code assets` and when recompiling it doesn't apply the changes? That's what I'm getting from this at least.

**Reply by ryi3r** [reply]
no i changed it from the `character data` folder

**Reply by Leather128** [reply]
Wait, did you change it from `character data/[your character name here]/` or `character data/`?

**Reply by ryi3r** [reply]
from `character data/<character name here>/`
